### PR TITLE
Draft: Part time ad salesperson

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: content/pages/jobs/ad-sales-part-time.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+You will work directly with the two of us and be able to have a big impact!
+
+To start with, this role would be part-time at about 20 hours per week.
+You will be helping with advertiser sales, reaching out to past advertisers about new campaigns,
+and help onboard new advertisers so their campaigns are a success.
+
+Our sales process is a bit different than some other places.
+Here are some important parts:
+
+- We believe that cold calling and cold outreach is a waste of time for our current business.
```
Comment (ericholscher) [inline content/pages/jobs/ad-sales-part-time.md:27]: This seems like an odd item to highlight as the first item in the list? I think we want to start with positives and excitement, not negatives.

File: content/pages/jobs/ad-sales-part-time.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  Your role will be to help get more of the latter type of advertiser
+  and help them become regular advertisers.
+- EthicalAds does not have long-term contracts lasting more than about a quarter.
+  This means that a lot of our business needs to be won every quarter on the merits
+  of the previous campaign. Selling hard and then under-delivering does not grow the business.
+  Winning repeat business is how our business actually grows.
+
+Interest and skills in any of the following is a big plus so please let us know in your application:
+
+- Advertising & digital marketing
```
Comment (ericholscher) [inline content/pages/jobs/ad-sales-part-time.md:43]: Presumably we want to mention sales skills as well? Ad sales? Inbound sales?

File: content/pages/jobs/ad-sales-part-time.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+> We aren't taking applications for this position yet. Stay tuned!
+
+
+## About the job
+
+Currently, every single advertiser talks to either Eric Holscher or David Fischer.
+You will be selling privacy-focused developer advertising to technical marketers.
+You will work directly with the two of us and be able to have a big impact!
```
Comment (ericholscher) [inline content/pages/jobs/ad-sales-part-time.md:18]: I think we want to mention that this is our first sales hire, and that we're definitely open to improving our sales process.

**Review by ericholscher — COMMENTED** [review]
Looks great overall, thanks for putting this together. Just a few nitpicks in-line, but I definitely think there's a bit more room here to be open to ideas and improvement, and not feel like we have the sales process completely figured out. The overview is super helpful, but I think we want to show a little bit of excitement for learning from them as well?

**Comment by davidfischer** [comment]
Just to be clear, I think there's whole sections of this post NOT present yet such as details about the company and about the team. This is *just* the part about the role. I'm not planning to merge this quite yet.

**Comment by davidfischer** [comment]
I'm going to close this for now. We aren't publicly posting this job.

**Pull request closed without merging**